Expensify/App
# Enable SVG image assets

cc @shawnborton

### Details
This enables the use of SVG assets in the Expensify.cash codebase.

One can pass a variable fill color to a SVG asset as is done in this PR, BUT:

1) It will only work with one-color assets
2) The assets themselves must not specify a fill color in the internals of the `svg`

This is because the fill passed in via JS is passed to the parent `svg` element, not individual `polygons`, `circles`, `paths`, etc...

We can work around this and gain finer control when necessary, by following this tutorial: https://blog.usejournal.com/how-to-use-svg-in-react-native-2-4140b146b7d1

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/149127
Fixes https://github.com/Expensify/Expensify.cash/issues/1088

### Tests
1) Run the app and sign in
1) Open an unpinned chat
1) Verify that the pin icon is visible and a dull gray
1) Tap or click the pin icon, and watch it become a charcoal
1) Tap the icon again, and watch it return to the dull gray it was before

### Tested On


### Screenshots

#### Web

##### Disabled
![image](https://user-images.githubusercontent.com/47436092/103324084-b8710b80-49fa-11eb-95f3-a2c72c0da3e4.png)

##### Enabled
![image](https://user-images.githubusercontent.com/47436092/103324102-cde63580-49fa-11eb-8675-74b354421943.png)

#### Mobile Web

##### Disabled
<img src="https://user-images.githubusercontent.com/47436092/103324162-02f28800-49fb-11eb-83f4-ca32c8ed8990.PNG" alt="" height="350px" />

##### Enabled
<img src="https://user-images.githubusercontent.com/47436092/103324152-fbcb7a00-49fa-11eb-8224-1d7f4e1156ed.PNG" alt="" height="350px" />

#### Desktop

##### Disabled
![image](https://user-images.githubusercontent.com/47436092/103324193-359c8080-49fb-11eb-9375-11f68564d635.png)

##### Enabled
![image](https://user-images.githubusercontent.com/47436092/103324198-3e8d5200-49fb-11eb-8952-c561d700a694.png)

#### iOS
https://user-images.githubusercontent.com/47436092/103324206-48af5080-49fb-11eb-9206-c0def9cbd70c.mov

#### Android

##### Disabled
![image](https://user-images.githubusercontent.com/47436092/103375177-57931300-4a8e-11eb-8757-910acfaef772.png)

##### Enabled
![image](https://user-images.githubusercontent.com/47436092/103375200-67aaf280-4a8e-11eb-999d-ddeeaf39875d.png)
